google-cloud-spanner/CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Release History
 
+### 1.11.0 / 2019-10-07
+
+#### BREAKING CHANGES (LOWER-LEVEL API)
+
+* Make the session_count argument required in the lower-level API batch_create_sessions call.
+
+#### Bug Fixes
```

Comment:
I would remove this section since it is covered in the breaking changes section above. (I don't think the change is really a bug fix either.)